docs/credits.md

```diff
@@ -1788,12 +1823,14 @@ Octopus Deploy is made possible thanks to the following great third-party produc
 | static-extend                                     | Jon Schlinkert                                                            | [https://github.com/jonschlinkert/static-extend](https://github.com/jonschlinkert/static-extend)                                                                                                                            | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | statuses                                          |                                                                           | [https://github.com/jshttp/statuses](https://github.com/jshttp/statuses)                                                                                                                                                    | [MIT](https://spdx.org/licenses/MIT.html)                                                                           |
 | stealthy-require                                  | Nicolai Kamenzky                                                          | [https://github.com/analog-nico/stealthy-require](https://github.com/analog-nico/stealthy-require)                                                                                                                          | [ISC](https://spdx.org/licenses/ISC.html)                                                                           |
+| step-validation                                   | Octopus Deploy                                                            | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
```

Comment:
@andrewabest Not sure if you're the best person, but trying anyhow: It appears this library is missing some of the information needed for licensing.